examples/12-fluids/00-fluids_model.py

```diff
@@ -0,0 +1,193 @@
+"""
+.. _ref_fluids_model:
+
+Explore Fluids models
+------------------------------------------------------
+
+"""
+
+###############################################################################
+# Exploring an Ansys Fluent model
```

Comment:
@rafacanton I fear that the introduction to this example should mention both CFX and FLuent right from the start, otherwise people interested in CFX may not take the time to scroll down to check if there is content they are interested in